BP/common/religion/religions/00_berber.txt

```diff
@@ -262,16 +271,41 @@
 				HighGodHerHis = CHARACTER_HERHIS_HIS
 			}
 
-			holy_site = fes
+			#WtWSMS: Swarty, Rome, Trapani, Carthage, Aneb
+			holy_site = ghat #Replaces Swarty; closest analog that can be found
+			holy_site = rome
+			holy_site = trapani
 			holy_site = carthage
-			holy_site = tinmallal
-			holy_site = sijilmasa
-			holy_site = wadan
+			holy_site = leptis_magna
+			
+			#Roman-Influenced Faith
+			doctrine = doctrine_theocracy_temporal
 
 			#Generic pagan tenets
 			doctrine = tenet_communal_identity
 			doctrine = tenet_ancestor_worship
 			doctrine = tenet_hellenism_syncretism
 		}
+		mastimani = {# Mastimani
+			color = { 0.5 0.05 0.0 }
+			icon = hellenic
+			reformed_icon = hellenic_reformed
+			doctrine = unreformed_faith_doctrine
+
+			#WtWSMS: Thessalonika, Athens, Abydos, Alexandria, Rome
+			#Converted to other Berber religion holy sites
+			holy_site = ghat #Replaces Swarty; closest analog that can be found
```

Comment:
Weren't you saying Swarty would be represented since it was a major city?

Reply:
I can't find where I said that; I thought I said "I can't find any references to Swarty or its locations from my searches". It might be an issue with conflicting romanizations though. Based on where Ghat is, I assume it's close to Swarty's location.

Reply:
Ok, I must have confused it then. But maybe it might make more sense to modify it to cities like Murzuk or Garama?

Reply:
Making it gerama as it was the Garamantian capital